README.md

```diff
@@ -135,3 +131,67 @@ See
+    <td align="center"><a href="http://mehari.me"><img src="https://avatars2.githubusercontent.com/u/9920926?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://github.com/mastermunj"><img src="https://avatars1.githubusercontent.com/u/968242?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://github.com/truongminh"><img src="https://avatars1.githubusercontent.com/u/3071451?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://github.com/joeytwiddle"><img src="https://avatars3.githubusercontent.com/u/911799?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://github.com/pktippa"><img src="https://avatars1.githubusercontent.com/u/7558577?v=4" width="100px;" alt=""/></a></td>
+  </tr>
+  <tr>
+    <td align="center"><a href="http://about.me/randmckinney"><img src="https://avatars2.githubusercontent.com/u/2925364?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://github.com/rashmihunt"><img src="https://avatars3.githubusercontent.com/u/19983365?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="http://www.sourcefuse.com"><img src="https://avatars2.githubusercontent.com/u/13620435?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://github.com/superkhau"><img src="https://avatars2.githubusercontent.com/u/1617364?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="http://www.virk.cc"><img src="https://avatars1.githubusercontent.com/u/3311536?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://github.com/Setogit"><img src="https://avatars3.githubusercontent.com/u/5251789?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://vvdwivedi.com"><img src="https://avatars2.githubusercontent.com/u/4401968?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="http://yashmurty.com"><img src="https://avatars0.githubusercontent.com/u/1630458?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://github.com/bschrammIBM"><img src="https://avatars2.githubusercontent.com/u/27782410?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://www.el-davo.com/"><img src="https://avatars0.githubusercontent.com/u/11195158?v=4" width="100px;" alt=""/></a></td>
+  </tr>
+  <tr>
+    <td align="center"><a href="https://medium.com/@haoxin"><img src="https://avatars2.githubusercontent.com/u/2569835?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://github.com/ibmjm"><img src="https://avatars0.githubusercontent.com/u/17479212?v=4" width="100px;" alt=""/></a></td>
+    <td align="center"><a href="https://github.com/activedecay"><img src="https://avatars3.githubusercontent.com/u/4298390?v=4" width="100px;" alt=""/></a></td>
+  </tr>
+</table>
+
+<!-- markdownlint-enable -->
+<!-- prettier-ignore-end -->
+
+<!-- ALL-CONTRIBUTORS-LIST:END -->
```

Comment:
I noticed that the code generated by `all-contributors generate` doesn't play well with Prettier (The extra newline on L188 is not there when generated). So I've added an extra step to the npm script to run Prettier against `README.md`.

I'm not sure if this is a good way to do it.. I'm open to suggestions.